### PR TITLE
[iOS] Apply SearchView Cell design

### LINKED ISSUE #604: [iOS] Apply SearchView Cell design
**Idea Description**

- Currently, SearchView's cell doesn't show session time and cancellation label. Please show session time and that label.

**Reference images and links**

スクリーンショット 2022-09-19 12 55 48

### PR DESCRIPTION
**Pull request by yamanetaisei**
## Issue
- close #604 

## Overview (Required)
- add session time label
- add cancel label

## Links
- 

## Screenshot
Before | After
:--: | :--:
 | 
||

### CODE REVIEW

File: app-ios/Sources/CommonComponents/TimetableListItemView.swift [hunk 1]
```diff
@@ -38,6 +38,14 @@ public struct TimetableListItemView: View {
...
                     }
                 }
+                if item.asSession()?.message != nil {
+                    HStack(spacing: 4) {
+                        Assets.error.swiftUIImage
+                        Text(StringsKt.shared.session_cancel.desc().localized())
+                            .foregroundColor(AssetColors.error.swiftUIColor)
+                            .font(Font.system(size: 12, weight: .regular, design: .default))
+                    }
+                }
```
Comment (ry-itto) [inline app-ios/Sources/CommonComponents/TimetableListItemView.swift:48]: How about combine with above `if let`?
example:

```swift
                if let session = item as? TimetableItem.Session,
                   session.speakers.isEmpty == false {
                    VStack(spacing: 0) {
                        HStack(spacing: 8) {
                            ForEach(session.speakers, id: \.self) { speaker in
                                PersonLabel(name: speaker.name, iconUrl: speaker.iconUrl)
                            }
                        }
                        if let message = session.message {
                            HStack(spacing: 4) {
                                Assets.error.swiftUIImage
                                Text(message...)
                                    .foregroundColor(AssetColors.error.swiftUIColor)
                                    .font(Font.system(size: 12, weight: .regular, design: .default))
                            }
                        }
                    }
                }
```

Comment (ry-itto) [reply app-ios/Sources/CommonComponents/TimetableListItemView.swift:48]: And I think `message` property may be useful.

File: app-ios/Sources/SearchFeature/SearchedSessionTimeView.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+
+struct SearchedSessionTimeView: View {
```
Comment (ry-itto) [inline app-ios/Sources/SearchFeature/SearchedSessionTimeView.swift:3]: How about create common component and use in Search and Timetable?
A component Inside `TimetableListView` has almost all the same as this.

File: app-ios/Sources/CommonComponents/TimetableListItemView.swift (hunk 1 above)
Comment (yamanetaisei) [reply app-ios/Sources/CommonComponents/TimetableListItemView.swift]: I agree !

[9d7f713](https://github.com/DroidKaigi/conference-app-2022/pull/654/commits/9d7f713fa402a86a468763554e613087b22755cd)

File: app-ios/Sources/SearchFeature/SearchedSessionTimeView.swift (hunk 2 above)
Comment (yamanetaisei) [reply app-ios/Sources/SearchFeature/SearchedSessionTimeView.swift]: I agree !

22b4bc6
2d0fb83

**Review by ry-itto — APPROVED** [review]
Looks Good! Thanks!

**Pull request merged**